tools/schemacode/schemacode/render.py

```diff
@@ -119,6 +121,9 @@ def make_glossary(schema):
         obj_desc = obj_desc.replace("\n\n", "<br>")
         # Otherwise a newline corresponds to a space
         obj_desc = obj_desc.replace("\n", " ")
+        # Spec internal links need to be replaced
+        if relpath is not None:
+            obj_desc = obj_desc.replace("PATH_TO_SRC", relpath)
```

Comment:
`PATH_TO_SRC` feels out of place in the glossary. Would `SPEC_ROOT` work for you?

```suggestion
            obj_desc = obj_desc.replace("SPEC_ROOT", relpath)
```

Reply:
I don't think it's all that bad given the files do live in `src`, is anyone going to be looking at this outside of the context of this repo?

Reply:
The metadata terms now have this string in their link URLs.

Reply:
fine with me :+1: fixed in https://github.com/bids-standard/bids-specification/pull/1056/commits/f609994687a3aee961ab4843bb2fee69b7cc2f2c